data-prepper-plugins/otel-proto-common/src/main/java/org/opensearch/dataprepper/plugins/otel/codec/OTelProtoCodec.java

```diff
@@ -92,6 +89,7 @@ public class OTelProtoCodec {
     static final String INSTRUMENTATION_LIBRARY_VERSION = "instrumentationLibrary.version";
     static final String STATUS_CODE = "status.code";
     static final String STATUS_MESSAGE = "status.message";
+    static final String ATTRIBUTES_KEY = "attributes";
```

Comment:
Shouldn't there be something like `instrumentationScope.attributes`? These attributes should be handled similar to resource, signal, or event attributes.